hl7/_from_hl7.py

```diff
@@ -70,7 +70,13 @@ def _convert(hl7_content: str) -> Iterator[Tuple[str, str, str, str]]:
             for index2, field2 in enumerate(field.split("^"), start=1):
                 if not field2:
                     continue
-                description = _get_description(segment_name, index1, index2)
+                if index2 == 1:
```

Comment:
Do we also want to do this for `index3` in the next function?

For instance: `3.5.1` becomes `3.5`?